### ISSUE TITLE
[Bug] when scale is negative , the output of fusedbiasleakyrelu is incorrect

### ISSUE THREAD

**Issue by jayggh** [open-mmlab/mmcv#2620]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmcv/issues) and [Discussions](https://github.com/open-mmlab/mmcv/discussions) but cannot get the expected help.
- [X] The bug has not been fixed in the latest version(https://github.com/open-mmlab/mmcv).

### Environment

None

### Reproduces the problem - code sample

import torch
import mmcv
from mmcv import ops
input =  torch.tensor([[1,2,1],[1,2,5]],dtype=float32,requires_grad=True).cuda()
bias = torch.tensor([0.1,0.2,0.3]).cuda()
output = ops.fused_bias_leakyrelu(input,bias,0.2,-1)
output.backward(torch.ones_like(output))
print(input.grad)

### Reproduces the problem - command or script

None

### Reproduces the problem - error message

when input+bias is postive, the gradient is scale * negative_slop(-0.2), but the correct gradient is -1.

**Comment by zhouzaida** [comment]
Hi, thanks for your feedback. The `scale` should not be less than 0.
